README.md

```diff
@@ -65,6 +64,20 @@ contentful space export --config ./__tests__/exports/contentful/config.json --sp
 npx contentful-cli space export --config ./__tests__/exports/contentful/config.json --space-id <your-contentful-space-id> --environment-id <your-contentful-environment-id>
 ```
 
+## Tests
+
+At this point, no further configuration is required to run the _**unit tests**_.
+```bash
+npm run test
+```
+
+More of the same, for the _**E2E**_ tests. Here it is crucial to have configured our `.env` and imported the dummy content to our space.
+```bash
+npm run test:e2e
+```
+
+The pipeline is configured to use the Contentful space managed by the _**agnostic.cms.harmonizer@proton.me**_ account during the _**E2E**_ tests execution. If the content of these tests needs to be updated in the provider, you must have access to this [vault](https://beyondsecure.onebeyond.cloud/vaults/OB-BpuUbMukgIKDSxGqmypk/secrets), otherwise, you can open an issue referencing the new content exported.
```

Comment:
@Bounteous17 , have you already added the credentials to the github repository secrets?

Reply:
@inigomarquinez yes, pipeline is actually working
https://github.com/onebeyond/agnostic-cms-harmonizer/actions/runs/8018272394